templates/pages/content.handlebars

```diff
@@ -42,7 +43,16 @@
 	  </div>
           <div class="flex-column">
             <div id="plugin-bug-report"></div>
-            <div id="plugin-rating"></div>
+            <bug-report
```

Comment:
Please name the component starting with vl- to reduce the possibilities of name collisions with other components

Reply:
sure sir, I'll update at both places.